netlify.toml

```diff
@@ -1,3 +1,2 @@
 [build]
-  ignore = "if  [ ${COMMIT_REF} == ${CACHED_COMMIT_REF} ] ; then false ; else git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF -- docs mkdocs.yml ; fi"
-
+ignore = "if  [ ${COMMIT_REF} == ${CACHED_COMMIT_REF} ] ; then false ; else git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF -- docs mkdocs.yml /packages/payment-proxy-client packages/nitro-rpc-client ; fi"
```

Comment:
it's awkward to have a single `netlify.toml` and ignore command for 3 separate sites which all use this repo :-/